.github/workflows/ci-macos.yml

```diff
@@ -0,0 +1,38 @@
+
+name: CI MacOS
+
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: macos-latest
+    
+    strategy:
+      matrix:
+        build_type: [Debug, Release]
+        cxx_compiler: [g++-11, clang++]
+        cxx_standard: [11]
```

Comment:
I don't know whether it's easier to duplicate this file or make the matrix of the old CI file bigger by extending it with MacOS x Clang with some `exclude`/`include` clauses https://github.com/nvim-treesitter/nvim-treesitter/blob/6227ae19b88061ec69770d7ef712a4d76c108acb/.github/workflows/test-queries.yml#L25-L49

Reply:
Seems to pass: https://github.com/theHamsta/Vulkan-Hpp/runs/6979116814?check_suite_focus=true

Reply:
also here 
![image](https://user-images.githubusercontent.com/7189118/174743818-2cec5c0a-36b4-4295-b07f-13803f418138.png)